src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -107,7 +107,7 @@ object RobotContainer {
         driverController().back()
             .whileTrue(gripper.setRollerPower(0.4))
             .onFalse(gripper.stop())
-        driverController().start().whileTrue(intake.reset())
+        driverController().start().whileTrue(intake.reset()).onFalse(intake.finishReset())
```

Comment:
Instead, `intake.reset()` can use `runEnd()` decorator.

Reply:
I'm not really sure what is the decorator you're referring to.
Also i think it doesn't matter that much.

Reply:
You mean runend command?

Reply:
Yes :)